contentcuration/contentcuration/frontend/channelEdit/components/edit/DetailsTabView.vue

```diff
@@ -642,10 +642,18 @@
         return (this.firstNode && this.getContentNodeFiles(this.firstNode.id)) || [];
       },
       fileDuration() {
+        let audioVideoFiles;
         if (this.firstNode.kind === 'audio' || this.firstNode.kind === 'video') {
-          return this.nodeFiles.filter(
+          audioVideoFiles = this.nodeFiles.filter(
             file => file.file_format === 'mp4' || file.file_format === 'mp3'
-          )[0].duration;
+          );
+          if (audioVideoFiles && audioVideoFiles[1]) {
```

Comment:
I don't quite understand the significance of the `[1]` indexing here, compared to the next if condition. Some code comments would help. Or, if it could be simplified to always use the _last file's duration_, then that would be:
```js
const audioVideoFile = audioVideoFiles.pop();
if (audioVideoFile) {
...
```

Reply:
Also worth noting, I'm not sure you need `if (audioVideoFiles` portion of the condition. The previous array filter will always return an array, which in JS, an empty array is always truthy (you can verify that with `Boolean([])`)